women-in-tech-resources/resources-list.md

```diff
@@ -58,6 +58,30 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 
 ### Mailing Lists, Google or Facebook Groups, and Slacks:
 - [LinuxChix India](http://india.linuxchix.org/)
+- [Facebook Developer Circle - Various States](https://developers.facebook.com/developercircles)
 
 ### Meetups and Events for Women & Non-binary Engineers:
 - [Women Who Code - Delhi](https://www.womenwhocode.com/delhi)
+- [Google Developers Group - Various States](https://developers.google.com/programs/community/)
```

Comment:
Are these groups free?

Reply:
yes. one can apply to join these groups as volunteers. They organize meetups, talks etc. One can know about current developments in tech and these are good for networking as well.

Reply:
As far as I know, Women Techmakers too fits in this category. Just a suggestion.

Reply:
noted. i will add it in a new pull request among others

Reply:
i have added the WTM India page link . I will add more resources in a new pull request